studies/replacebg.py

```
@@ -6,6 +6,13 @@
 import os
 from src import pandas_helper, logger
 
+def get_basal_df_mock():
+    df = pd.DataFrame(columns=['PtID', 'datetime', 'DeviceDtTmDaysFromEnroll', 'DeviceTm', 'Duration',
+                          'ExpectedDuration', 'SuprDuration', 'RecID', 'Rate'])
+    df['Rate'] = pd.to_numeric(df['Rate'])
+    return df
+
+
```

same as before, let's throw an error when load_data() fails.